features/features.go

```diff
@@ -119,6 +119,13 @@ type Config struct {
 	// unique "INSERT ... RETURNING" functionality.
 	InsertAuthzsIndividually bool
 
+	// AutomaticallyPauseZombieClients configures the RA to automatically track
+	// limiter to be the authoritative source of rate limiting information for
+	// automatically pausing clients who systemically fail every validation
+	// attempt. When disabled, only manually paused accountID:identifier pairs
```

Comment:
I can't parse this sentence. Specifically "to automatically track limiter to be the authoritative source ..." seems to be an editing error?

Reply:
Weird this looks like some kind of hybrid of what was there and what I put in my suggestion: https://github.com/letsencrypt/boulder/pull/7763#discussion_r1834648954